cdap-ui/app/cdap/components/Experiments/CreateView/index.js

```diff
@@ -67,8 +67,8 @@ export default class ExperimentCreateView extends Component {
     experimentId: createExperimentStore.getState().experiments_create.name,
     isSplitFinalized: createExperimentStore.getState().model_create.isSplitFinalized,
     loading: createExperimentStore.getState().experiments_create.loading,
-    active_step: createExperimentStore.getState().active_step.step_name,
-    redirectToExperimentDetail: false
+    step_name: createExperimentStore.getState().active_step.step_name,
```

Comment:
nit: I would advice having the `active_step` as step_name doesn't signify what it indicates. We actually store the active step in that state variable.